packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -44,57 +44,204 @@ const isDropdownSelect = (element = {}) => {
 
 export default class Calendar extends React.Component {
   static propTypes = {
+    /**
+     * Prop to change date when a valid date is selected.
+     */
     adjustDateOnChange: PropTypes.bool,
+    /**
+     * Class name to style the date picker.
+     */
     className: PropTypes.string,
+    /**
+     * Components to render within date picker.
+     */
     children: PropTypes.node,
+    /**
+     * Format of the date selected.
+     */
     dateFormat: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.array
     ]).isRequired,
+    /**
+     * Prop to style individual days on calendar.
+     */
     dayClassName: PropTypes.func,
+    /**
+     * Prop to select a dropdown mode for the year dropdown.
+     */
     dropdownMode: PropTypes.oneOf(['scroll', 'select']).isRequired,
+    /**
+     * Maximum Date for a given range.
+     */
     endDate: PropTypes.object,
+    /**
+     * Array to exclude certain dates.
+     */
     excludeDates: PropTypes.array,
+    /**
+     * A callback function to be executed when filter dates is triggered.
+     */
     filterDate: PropTypes.func,
+    /**
+     * Specifies whether the height of calendar dom fixed or variable.
+     */
     fixedHeight: PropTypes.bool,
+    /**
+     * A callback function to be executed to format week number .
+     */
     formatWeekNumber: PropTypes.func,
+    /**
+     * Highlight range of dates with custom classes.
+     */
     highlightDates: PropTypes.instanceOf(Map),
+    /**
+     * Show dates only in the given array.
+     */
     includeDates: PropTypes.array,
+    /**
+     * Prop to show inline version of date picker component.
+     */
     inline: PropTypes.bool,
+    /**
+     * Name of locale data for different international formatting.
+     */
     locale: PropTypes.string,
+    /**
+     * Maximum Value of date that can be selected by user.
```

Comment:
nit: lower case v in Value. Also in minDate description.

Reply:
Resolved in 1b98d9f